service/src/geopm_hash.c

```diff
@@ -43,7 +47,19 @@ extern "C"
 
 uint64_t geopm_crc32_u64(uint64_t begin, uint64_t key)
```

Comment:
Why does the Agg class have a dependency on this thing?  If this is a feature of the runtime, it should not be in the service directory.

Reply:
It doesn't, but many of the tests in the service/test/*.cpp include the geopm_hash.h header.